components/notability/commons/notability_checker.lua

```diff
@@ -245,7 +245,7 @@ end
 function NotabilityChecker._calculateDateLoss(date)
 	local timestamp = _lang:formatDate('U', date)
 	local differenceSeconds = _NOW - timestamp
-    return math.floor(differenceSeconds / _SECONDS_IN_YEAR) + 1
+	return math.floor(differenceSeconds / _SECONDS_IN_YEAR) + 1
```

Comment:
Wasn't this changed in #892?

Reply:
That's just from the merge resolve
Looking at the full diff it doesn't show up
https://github.com/Liquipedia/Lua-Modules/pull/893/files

Reply:
Yes, please rebase instead.